packages/renderer/src/lib/image/LoadImages.svelte

```diff
@@ -0,0 +1,145 @@
+  const archives = await window.openDialog({
+    title: 'Select Tar Archive(s) containing Image(s) to load',
+    selectors: ['multiSelections'],
+  });
+
+  if (!archives) {
+    return;
+  }
+
+  archivesToLoad = [...archivesToLoad, ...archives];
+}
+
+function deleteImagesTarArchiveToLoad(index: number) {
+  archivesToLoad = archivesToLoad.filter((_, i) => i !== index);
+}
+
+async function loadImages() {
+  loadError = '';
+
+  if (!selectedProvider) {
+    throw new Error('Select a provider to load');
+  }
+
+  inProgress = true;
+
+  const task = createTask('Load images');
+
+  for (const archive of archivesToLoad) {
+    try {
+      await window.loadImages({
+        provider: selectedProvider,
+        archives: [archive],
+      });
+    } catch (e) {
+      loadError += `Error while loading ${archive}: ${String(e)}\n`;
+    }
+  }
+
+  inProgress = false;
+  if (loadError === '') {
+    task.status = 'success';
+    task.state = 'completed';
+    router.goto('/images');
+  } else {
+    task.status = 'failure';
+    task.error = loadError;
+    task.state = 'completed';
+  }
+}
+</script>
+
+<FormPage title="Load Images">
+  <svelte:fragment slot="icon">
+    <i class="fas fa-play fa-2x" aria-hidden="true"></i>
+  </svelte:fragment>
+  <div slot="content" class="p-5 min-w-full h-fit">
+    <div class="bg-charcoal-600 px-6 py-4 space-y-2 lg:px-8 sm:pb-6 xl:pb-8">
+      {#if providerConnections.length > 1}
+        <label for="providerChoice" class="pt-6 block text-sm font-bold text-gray-400"
+          >Container Engine
+          <select
+            class="w-full p-2 outline-none text-sm bg-charcoal-600 rounded-sm text-gray-700 placeholder-gray-700"
+            name="providerChoice"
+            id="providerChoice"
+            bind:value="{selectedProvider}">
+            {#each providerConnections as providerConnection}
+              <option value="{providerConnection}">{providerConnection.name}</option>
+            {/each}
+          </select>
+        </label>
+      {/if}
+
+      <label for="modalImagesLoad" class="pt-4 block mb-2 text-sm font-medium text-gray-400"
+        >Images archive to load:</label>
+      <Button on:click="{addArchivesToLoad}" icon="{faPlusCircle}" type="link">Add archive</Button>
+      <!-- Display the list of existing imagesToLoad -->
+      {#if archivesToLoad.length > 0}
+        <div class="flex flex-row justify-center w-full py-1 text-sm font-medium text-gray-400">
+          <div class="flex flex-col grow pl-2">Archive</div>
+        </div>
+      {/if}
+      {#each archivesToLoad as archiveToLoad, index}
+        <div class="flex flex-row justify-center w-full py-1">
+          <Input bind:value="{archiveToLoad}" aria-label="archive path" readonly="{true}" />
+          <Button type="link" on:click="{() => deleteImagesTarArchiveToLoad(index)}" icon="{faMinusCircle}" />
+        </div>
+      {/each}
+
+      <div class="pt-5 border-zinc-600 border-t-2"></div>
```

Comment:
I thought the Add Archive button was in an input field the first time I saw the screenshot. We don't really use dividers elsewhere so I would just remove, but if not there needs to be more spacing.